Topics/statistics-fundamentals.md

```diff
@@ -0,0 +1,167 @@
+- [The Best Stats You've Ever Seen](https://www.ted.com/talks/hans_rosling_shows_the_best_stats_you_ve_ever_seen)
+
+---
+
+To understand the concepts of descriptive statistics, the book Virgilio recommends is the following:
+
+[**Statistics in Plain English**](http://www.pindex.com/uploads/post_docs/statistics%20in%20plain%20english%20(3rd,%202010)(PINDEX-DOC-6952).pdf) - [Timothy C. Urdan](https://www.amazon.com/Timothy-C.-Urdan/e/B001JP8H9Q%3Fref=dbs_a_mng_rwt_scns_share)
+
+This book is fantastic because it focuses more on **the reason behind the concepts** concepts rather than on formulas or calculations, which are easily found elsewhere. 
+
+**The book is mandatory for a full understanding of descriptive statistics** (it is certainly not the only book of statistics out there, but it is the best in terms of clarity of explanation and completeness of the basic concepts.).
+
+Once finished, you can choose whether to go deeper with a more technical manual, like the following _free book_:
+
+[**Elements of Statistics**](https://www.open.edu/openlearncreate/mod/oucontent/view.php?id=18263&printable=1)
+
+This book deals in more depth with the various topics, but to start doing small experiments in Data Science is not necessary to address it immediately. 
+
+**Keep it as a reference!**
+
+#### Check your knowledge
+
+Test your knowledge with some of [these exercises](https://www0.gsb.columbia.edu/faculty/pglasserman/B6014/exercises.pdf).
+
+Anyway, you can find a ton of them online, and remember, that the more you practice, the better you will get!
+
+
+
+### Probability Theory
+
+Probability theory governs every daily event, from microscopic to macroscopic, even if we hardly realize it!
+
+But you may have thought about the odds of winning, for example, the game of dice, or when you estimate the probability of having successfully passed an exam!
+
+Probability theory is so interconnected with statistical and mathematical concepts that it _is difficult to explain why it is fundamental_: simply consider it as the basic building blocks of more advanced statistical concepts based on it.
+
+Not to mention that knowledge of probability theory can be very useful in everyday life since it can suggest to us how to behave on certain occasions, to make the best choices, or involving the least risk of failure.
+
+Fortunately, you can have the (perhaps) best course on probability theory available to date, offered as usual by MIT. 
+
+It's good to have a first-class education from home for free, huh? 
+
+What an incredible time to be alive!
+
+Take the course at:
+
+[**Introduction to Probability - Video Lectures**](https://www.youtube.com/watch?v=1uW3qMFA9Ho&list=PLUl4u3cNGP60hI9ATjSFgLZpbNJ7myAg6)
+
+In addition, keep this _free complete book_ as a reference:
+
+[**Introduction to Probability - Book**](https://www.dartmouth.edu/~chance/teaching_aids/books_articles/probability_book/amsbook.mac.pdf)
+
```

Comment:
https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/details/maths-stats-probability.md#probability-and-probabilistic-programming --- if you like to add this in some creative way